app/views/layouts/admin_layout.html.erb

```diff
@@ -22,7 +22,7 @@
         text: "Specialist sectors",
         href: topics_path,
         active: active_navigation_item == 'topics' ? 'active' : false,
-        user_has_permission_to_see: !livestream_editor?
+        user_has_permission_to_see: true
```

Comment:
Minor, it looks like the convention that already exists is to remove this argument when there aren't any permissions involved (see https://github.com/alphagov/collections-publisher/blob/c8676a3ef2565d1917cf73418f1863ea3c438be0/app/views/layouts/admin_layout.html.erb#L39-L40)

Reply:
Thanks! Will update and fix up.